lighthouse-core/lib/traces/tracing-processor.js

```diff
@@ -264,14 +255,23 @@ class TraceProcessor {
   }
 
   /**
+   * Identifying tasks from a trace is fun!
+   * - In <= m65, tasks were named `TaskQueueManager::ProcessTaskFromWorkQueue`
+   * - In m66-68, `ThreadControllerImpl::DoWork`, https://crrev.com/c/883346
+   * - In m69-70, DoWork changed definitions and `ThreadControllerImpl::RunTask` became our event, https://bugs.chromium.org/p/chromium/issues/detail?id=871204#c11
+   * - In m71+, We added RunTask to `disabled-by-default-lighthouse`
+   * - In m74+ RunTask moved to the `devtools-timeline` category, making the LH cat moot, http://crrev.com/630929
+   * - Soon after, in http://crrev.com/642705 RunTask moved to `disabled-by-default-devtools.timeline`
+   * and regressed as didn't catch top-level events inside microtasks,
+   * - This was identified & fixed in http://crbug.com/953914. The fix made it to m75 before stable release.
+   *
+   * In summary: as of m75, `cat: 'disabled-by-default-devtools.timeline', name: 'RunTask'` events are top-level tasks.
+   *
    * @param {LH.TraceEvent} evt
    * @return {boolean}
    */
   static isScheduleableTask(evt) {
-    return evt.name === SCHEDULABLE_TASK_TITLE_LH ||
```

Comment:
Can we leave this in here? Especially for tracehouse parsing older traces still has value and isn't *that* big of a maintenance burden.

Reply:
aiight.